cmd/ddev/cmd/config_test.go

```diff
@@ -514,6 +514,8 @@ func TestConfigDatabaseVersion(t *testing.T) {
 	// add a database into the config and nothing else
 	for _, dbTypeVersion := range versionsToTest {
 		_ = app.Stop(true, false)
+		_, err := exec.RunHostCommand(DdevBin, "delete", "-Oy", t.Name())
```

Comment:
I'm not sure how this does any harm, but what did you hit here that made you do this? Is it related to this PR? Did some old project get left laying around?

Reply:
See https://github.com/ddev/ddev/pull/5926#issuecomment-2013819550

Reply:
Could you please revisit this without the delete here to see if you can find out what the real problem is? It appears that you're having to patch something that is caused by something entirely different.  This kind of adaptation of a test is really suspicious, as you know.

Reply:
It looks to me like this code is just trying to call `ddev config` inside a directory which already has a ddev project created and that this should have been failing previously

Reply:
Except... it doesn't seem to have failed except in this PR. Thanks!

Reply:
Yup. The bit of code I mentioned in https://github.com/ddev/ddev/pull/5926#issuecomment-2013819550 is being removed in this PR - if we keep that there it still works, because there's no projects in the global config list and there _are_ projects in the new projects list. That discrepancy combined with the mentioned code results in clearing the new project list, so as far as DDEV is concerned there's no project there.

This PR removes that code, which means DDEV is now fully aware that there *is* a project there, causing the error.

I'm not sure if this was something introduced when I added the new project list file, or since then, or maybe I'm just wrong about the cause here? But that's what it looks like to me.

Reply:
Thanks for chasing it. I'm often unable to follow your references, too many context switches.  If you could re-mention things instead of linking to other comments maybe I'll do better. It sounds like you're saying that another test has left a project around instead of doing an `app.Stop(true, false)` or something.

Reply:
Not another test - this same test. It's running `ddev config` in a loop in the same directory. It _is_ doing `app.Stop(true, false)` but that seems to be insufficient to remove the project from the project list file.

Reply:
If app.Stop(true,false) isn't removing the project from the project list file, it's an introduced bug. Every single test would fail if that were common. Please step through in debugger and figure out what's going on. Thanks!

Reply:
Good to know - I'll introduce a new test that explicitly tests `app.Stop(true,false)` and debug it.
I'll try to find time this week to do this.